src/components/Delegations.js

```diff
@@ -236,6 +260,13 @@ class Delegations extends React.Component {
     return rewards.reward.find(reward => reward.denom === this.props.network.denom)
   }
 
+  calculateApy(validatorCommission, periodPerYear) {
```

Comment:
you need to bonding ratio in the calculation ?

APR = (inflation * (1 - (community tax / 100)) / (bonded tokens / total supply)

but some chain use different calculation (osmosis, juno, evmos)

Reply:
Can you help me with osmosis/Juno/evmos formula please ? (I speak French too)

Reply:
> Can you help me with osmosis/Juno/evmos formula please ? (I speak French too)

osmosis and evmos are epoch based. You need to evaluate the inflation before dividing by the bonded ratio.
Juno is based on the annual provision and block per year.

I will talk with my team, maybe we can provide a dedicated endpoint.

Reply:
any chance we can discuss on github about this, I'm in the ecostake discord as mateo | Stake Frites (🥩 , 🍟)#6208